tests/Hal/Application/Formater/Summary/ViolationsXmlTest.php

```diff
@@ -18,18 +18,18 @@ public function testFormaterReturnsXml() {
 
         // all results
         $rs1 = $this->getMockBuilder('\Hal\Component\Result\ResultSet')->disableOriginalConstructor()->getMock();
-        $rs1->expects($this->any())->method('asArray')->will($this->returnValue(array('volume' => 5)));
+        $rs1->expects($this->any())->method('asArray')->will($this->returnValue(array('loc' => 5)));
         $rs1->expects($this->any())->method('getFilename')->will($this->returnValue('path2/file1.php'));
         $rs2 = $this->getMockBuilder('\Hal\Component\Result\ResultSet')->disableOriginalConstructor()->getMock();
-        $rs2->expects($this->any())->method('asArray')->will($this->returnValue(array('volume' => 15)));
+        $rs2->expects($this->any())->method('asArray')->will($this->returnValue(array('loc' => 15)));
         $rs2->expects($this->any())->method('getFilename')->will($this->returnValue('path1/file1.php'));
 
         $collection = $this->getMockBuilder('\Hal\Component\Result\ResultCollection') ->disableOriginalConstructor() ->getMock();
         $collection->expects($this->any()) ->method('getIterator') ->will($this->returnValue(new \ArrayIterator(array($rs1, $rs2))));
         $collection->expects($this->any()) ->method('getFilename') ->will($this->returnValue('abc'));
         $collection->expects($this->any()) ->method('asArray') ->will($this->returnValue(array(
-            array('volume' => 5)
-            , array('volume' => 15)
+            array('loc' => 5)
+            , array('loc' => 15)
```

Comment:
I don't remember quite well but I think that's because the index ```loc``` is requested somewhere (?). So it is just to make things easier. I can look into this latter today.